.travis.yml

```diff
@@ -29,15 +29,41 @@ env:
     - SEQ_PG_PW=sequelize_test
     - SEQ_PG_HOST=127.0.0.1
     - SEQ_PG_PORT=8998
+    # oracle info
+    - NODE_ORACLEDB_HOST: localhost
+
+before_install:
+  - |
+    if [ $ORACLE_VER ]; then 
+      mkdir -p /opt/oracle && 
+      cd /opt/oracle && 
+      wget https://s3.amazonaws.com/sequelize/instantclient-basic-linux.x64-12.1.0.2.0.zip &&
+      wget https://s3.amazonaws.com/sequelize/instantclient-sdk-linux.x64-12.1.0.2.0.zip &&
+      unzip instantclient-basic-linux.x64-12.1.0.2.0.zip &&
+      unzip instantclient-sdk-linux.x64-12.1.0.2.0.zip &&
+      mv instantclient_12_1 instantclient &&
+      cd instantclient && 
+      ln -s libclntsh.so.12.1 libclntsh.so &&
+      export LD_LIBRARY_PATH=/opt/oracle/instantclient:$LD_LIBRARY_PATH &&
+      cd $TRAVIS_BUILD_DIR;
+    fi
 
 before_script:
   # mount ramdisk
+  - |
+    if [ $ORACLE_VER ]; then
+    wget https://s3.amazonaws.com/sequelize/data.tar.gz
```

Comment:
I would prefer if this was not hosted on my amazon account once this is merged - Since it's 2.6 gigabytes, so it's going to rack up a bit of transfer charges.

The client and SDK are fine, but this file is a bit too big ;)

Reply:
Maybe we could simply push an image to docker hub, its free and maybe travis caches images so it chould be also faster in the future 🤔

Reply:
That would be ideal - However the normal image requires a long (!!) time to initialize the db (+1 minute) - That's what we're trying to avoid by having the DB pre-initialized.

I'm guessing that could be integrated into a docker image, I just couldn't figure out how

Reply:
Oh, didn't know that, thats long! 
If someone needs help with the docker thing, just ping me, I could try that

Reply:
I've tried many things, but to push on docker hub a fully initialized image, but without success.
Fell free to test, any help on this is really appreciate :)
The repository is here https://github.com/konnecteam/docker-oracle-12c

Reply:
https://github.com/konnecteam/docker-oracle-12c/pull/1 there you go :)